DataToString.swift

```diff
@@ -129,10 +151,26 @@ func makeStringB(data: Data) -> String {
   } ?? ""
 }
 
+func dataToString(data: Data) -> String {
+    let holder = Holder(data: data)
```

Comment:
Need to either add a condition here that only goes down this path if the Data's encoding is ASCII / UTF8 (similar to Philippe's code).  The condition might belong in here, or rename this function to indicate that it is specifically an ASCII fastpath.